tests/search/search_export_tests.py

```diff
@@ -46,3 +137,24 @@ def test_login_via_basic_auth_invalid(self):
         response = SearchExport().get(request)
         self.assertEqual(request.user.username, "anonymous")
         self.assertEqual(response.status_code, HTTPStatus.UNAUTHORIZED)
+
+def get_response_json(client, tiles=True, export=True, format='tilecsv', report_link=False, **kwargs):
+    query = {
+        "tiles": tiles,
+        "export": export,
+        "format": format,
+        "report_link": report_link
+    }
+    for key in kwargs:
+        query[key] = kwargs.get(key, None)
+    for k, v in list(query.items()):
+        query[k] = JSONSerializer().serialize(v)
+    resource_reviewer_group = Group.objects.get(name="Resource Exporter")
+    test_user = User.objects.get(username="unpriviliged_user")
+    test_user.groups.add(resource_reviewer_group)
+    client.login(username="unpriviliged_user", password="test")
+    response = client.get("/search/resources", query)
+    response_json = json.loads(response.content)
+    return response_json
```

Comment:
My inclination would be to avoid checking this in to the repo until we need it.

Reply:
rm in latest